start

```diff
@@ -45,18 +61,72 @@ cat >> ${PG_CONFDIR}/pg_hba.conf <<EOF
 host    all             all             0.0.0.0/0               md5
 EOF
 
+# allow replication of the database
+if [ -n "${REP_USER}" ]; then
+  if [ "${PSQL_SSLMODE}" == "disable" ]; then
+    cat >> ${PG_CONFDIR}/pg_hba.conf <<EOF
+host    replication     $REP_USER       0.0.0.0/0               md5
+EOF
+    else 
+    cat >> ${PG_CONFDIR}/pg_hba.conf <<EOF
+hostssl replication     $REP_USER       0.0.0.0/0               md5
+EOF
+  fi
+fi
+
+
+# allow passwordless connections from current user
+if [ -n "${DB_USER}" ]; then
+  if [ ! -z "${DB_PASS}" ]; then
+    cat >> ~/.pgpass <<EOF
```

Comment:
@Downchuck If this change is not related to adding replication support, then I request you to submit an separate PR with this change. It will allow me to merge such changes immediately and/or initiate a separate discussion regarding it.Try and limit your PR on only to the replication feature. I hope you understand.